backend/app/services/crucible_readme.md

```diff
@@ -48,6 +48,7 @@ iteration number of a specific run.
 <dt>METRIC_DESC</dt><dd>this contains descriptive data about a specific series
     of metric values within a specific period of a run, including the metric UUID,
     the metric "class", type, and source, along with a set of "names" (key/value
+<<<<<<< HEAD
```

Comment:
An unresolved merge conflict here! (And several others in this file.) I think you should just take the readme file from `main` rather than try to merge the old version from the ILAB branches against the latest.